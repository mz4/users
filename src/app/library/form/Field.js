import React from 'react';
import styles from './Field.module.scss';

export const Field = ({ children }) => {
  return <div className={styles.field}>{children}</div>;
};
