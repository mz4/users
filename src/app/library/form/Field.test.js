import renderer from 'react-test-renderer';
import { render } from '@testing-library/react';
import { Field } from './Field';

describe('Render field component', () => {
  it('Create a snapshot', () => {
    const field = <Field />;
    const component = renderer.create(field);
    expect(component.toJSON()).toMatchSnapshot();
  });
  it('Render with props', () => {
    const field = (
      <Field>
        <div className={'testClass'} />
      </Field>
    );
    const { container } = render(field);
    const element = container.getElementsByClassName('testClass');
    expect(element).toBeDefined();
  });
});
