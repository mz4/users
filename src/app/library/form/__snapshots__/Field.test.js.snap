// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render field component Create a snapshot 1`] = `
<div
  className="field"
/>
`;
